### PR TITLE
Netflix data video

### PR DESCRIPTION
**Pull request by mrjoshida**
Add How Computers Work Series and Netflix data video

### CODE REVIEW

File: dashboard/config/videos.csv [hunk 1]
```diff
@@ -121,10 +121,15 @@ tutorial_puzzle_challenge,,,_l68chtl1co,http://videos.code.org/2014/tutorial-puz
 infinity_playlab_intro,,,vQGOfhsb76o,http://videos.code.org/2015/infinityplaylab/infinity-playlab-intro.mp4
 infinity_playlab_events,,,HcljJx_0Elw,http://videos.code.org/2015/infinityplaylab/infinity-playlab-events.mp4
 infinity_playlab_repeat_forever,,,Ldj1-kEa9nA,http://videos.code.org/2015/infinityplaylab/infinity-playlab-repeat-forever.mp4
-csd_concept_computer,,,f7WfMGA4u_w,http://videos.code.org/cs-discoveries/concept/what_makes_a_computer.mp4
-csd_concept_binary_data,,,NCi7BQ6OiCE,http://videos.code.org/cs-discoveries/concept/binary_and_data.mp4
+csd_concept_how_computers_work,,,OAx_6-wdslM,http://videos.code.org/cs-discoveries/concept/01_HowComputersWork_sm.mp4
+csd_concept_computer,,,mCq8-xTH7jA,http://videos.code.org/cs-discoveries/concept/02_WhatMakesAComputer_sm.mp4
```
Comment (caleybrock) [inline dashboard/config/videos.csv:125]: csd_concept_computer and csd_concept_circuits_logic are meant to have new videos, but not new titles?

**Comment by caleybrock** [comment]
How Computers Work series seems pretty high profile, are show notes on their way? 
I think it's fine to have them in separate PRs, but we should have them.

**Comment by caleybrock** [comment]
cc @breville (dotd) you'll probably need to scoop thumbnails on staging once this gets merged so that your eyes test run works

File: dashboard/config/videos.csv (hunk 1 above)
Comment (mrjoshida) [reply dashboard/config/videos.csv:125]: Yup, we had draft versions up for use in the curriculum.

**Comment by mrjoshida** [comment]
Show notes will come later, this is just so we can stage the videos into the curriculum

**Pull request merged**